package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "ToDo-List-redo",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack serve --open",
+    "build": "webpack"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/elyor-doniyorov/ToDo-List-redo.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/elyor-doniyorov/ToDo-List-redo/issues"
+  },
+  "homepage": "https://github.com/elyor-doniyorov/ToDo-List-redo#readme",
+  "devDependencies": {
+    "css-loader": "^6.2.0",
+    "html-webpack-plugin": "^5.3.2",
+    "style-loader": "^3.2.1",
+    "webpack": "^5.46.0",
+    "webpack-cli": "^4.7.2"
```

Comment:
Let's improve the webpack setup.
For using 'serve' command you need to install: 'webpack-dev-server' package.